react-native-gutenberg-bridge/ios/RNReactNativeGutenbergBridge.swift

```diff
@@ -45,15 +43,25 @@ public class RNReactNativeGutenbergBridge: RCTEventEmitter {
             })
         }
     }
-    
+
     @objc
     func requestOtherMediaPickFrom(_ source: String, allowMultipleSelection: Bool, callback: @escaping RCTResponseSenderBlock) {
-        //TODO implement me
+        requestMediaPickFrom(source, filter: nil, allowMultipleSelection: allowMultipleSelection, callback: callback)
     }
```

Comment:
This event (`requestOtherMediaPickFrom`) is probably not needed? We could reuse `requestMediaPickFrom`.

Reply:
Made this mentioned change here: https://github.com/WordPress/gutenberg/pull/18303